### DISCUSSION TITLE
Using apply_redactions return a blank page

### DISCUSSION THREAD

**Discussion by tcourat** [pymupdf/PyMuPDF#4064] [Q&A] [answered]
Hello

I am currently trying to split a single big pdf into hundreds of smaller pdf, where each pdf corresponds to a section.

I have been able to find the start of each section with the `get_toc` method, so I have the starting page and ending page. However when a section start in a middle of a page (or ends), I'd like to erase the content of the previous (or next) section that still appear on the starting (or ending) page of the section.

I managed to get the bouding boxes of the content I want to erase, but it returns a completly blank page despite the "red cross" preview from `add_redact_annot` shows its not covering the whole page.

AN EXAMPLE :
[section_248_Tree.pdf](https://github.com/user-attachments/files/17817306/section_248_Tree.pdf)

This returns two blank pages when I use `apply_redactions`.

Any advice ?

**Comment by JorjMcKie** [comment]
I should work and it does work with version 1.24.13.

**Reply by tcourat** [reply]
I am also using version 1.24.13

**Reply by tcourat** [reply, marked as answer]
Ok it seems that it is a bug from the vscode pdf viewer, because when I look in adobe acrobat it seems to work. Very strange...
